NSrtm.Core/Pgm/DataDesciption/PgmDataDescriptionExtractor.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace NSrtm.Core.Pgm
+{
+    internal static class PgmDataDescriptionExtractor
+    {
+        private static readonly Regex _preambleParser =
+            new Regex(
```

Comment:
While single big regex for all possible lines and not multiple regexes for each required line?